README.md

```diff
@@ -93,22 +93,68 @@ the pool and error is logged.
 Configuration is read from a YAML configuration file with the following fields:
 
 ```yaml
-# Listen interface and port e.g. "127.0.0.1:9090", ":80"
-Listen: ":8080"
-# Technical endpoint interface
-TechnicalEndpointListen: ":8071"
-# Technical health check endpoint (for load balancers)
-HealthCheckEndpoint: "/status/ping"
-# Additional not AWS S3 specific headers proxy will add to original request
-AdditionalRequestHeaders:
-    'Cache-Control': "public, s-maxage=600, max-age=600"
-    'X-Akubra-Version': '0.9.26'
-# Additional headers added to backend response
-AdditionalResponseHeaders:
-    'Access-Control-Allow-Origin': "*"
-    'Access-Control-Allow-Credentials': "true"
-    'Access-Control-Allow-Methods': "GET, POST, OPTIONS"
-    'Access-Control-Allow-Headers': "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type"
+Service:
+  Server:
+    BodyMaxSize: 100MB
+    MaxConcurrentRequests: 200
+    # Listen interface and port e.g. "0:8000", "localhost:9090", ":80"
+    Listen: ":7082"
+    # Technical endpoint interface
+    TechnicalEndpointListen: ":7005"
+    # Health check endpoint (for load balancers)
+    HealthCheckEndpoint: "/status/ping"
+  Client:
+    # Additional not AWS S3 specific headers proxy will add to original request
+    AdditionalResponseHeaders:
+        'Access-Control-Allow-Origin': "*"
+        'Access-Control-Allow-Credentials': "true"
+        'Access-Control-Allow-Methods': "GET, POST, OPTIONS"
+        'Access-Control-Allow-Headers': "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,X-CSRFToken"
+        'Cache-Control': "public, s-maxage=600, max-age=600"
+    # Additional headers added to backend response
+    AdditionalRequestHeaders:
+        'Cache-Control': "public, s-maxage=600, max-age=600"
+    # Backends in maintenance mode
+    # MaintainedBackends:
+    #  - "http://s3.dc2.internal"
+    # List request methods to be logged in synclog in case of backend failure
+    SyncLogMethods:
+      - GET
+      - PUT
+      - DELETE
+    # behaviours in application with HTTP protocol condition for better transmission
+    # DefaultTransport (with empty Triggers) it's only one required item in Transports section (logical validator is implemented)
+    Transports:
+      -
+        Name: Transport1
+        Triggers:
+          Method: GET|POST
+          Path: .*
```

Comment:
when I provide an empty `Path`, `Name`, or `Method` akubra starts, but crashes on serving request.
when I provide a following config 
```Transports:
      -
        Name:
        Triggers:
          Method:
          Path:
        Details:
          MaxIdleConns: 200
          MaxIdleConnsPerHost: 1000
          IdleConnTimeout: 2s
          ResponseHeaderTimeout: 5s
```
It starts and servers requests, even tho no proper transport is defined. Why?